evalai/utils/submissions.py

```diff
@@ -264,17 +264,17 @@ def display_submission_details(submission_id):
     pretty_print_submission_details(response)
 
 
-def display_submission_result(submission_id):
+def display_submission_with(submission_id, file_name, error_message):
```

Comment:
What are your opinions on changing `display_submission_with` -> `display_submission_file_data`

Reply:
Nice, I'll take that. Thanks for the suggestion.

Reply:
Done :+1: